src/Components/Scenes/Hydration/NewDestination/NewDestination-Form.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
```

Comment:
I'm not sure how the form is displaying on your screen but the inputs are slightly not aligned up in a way. Is this how it looks on your screen?

<img width="1364" alt="Screen Shot 2020-03-06 at 10 15 27 AM" src="https://user-images.githubusercontent.com/36801809/76110530-87b77580-5f93-11ea-877c-f4180395e374.png">

Reply:
dunno if you meant to fix that or just as a prototype though